data/Makefile

```
@@ -696,6 +696,10 @@ minsize=2000
 %.bic.tsv: %.tsv
 	$(python) $(bin)/physlr biconnected-components $< >$@
 
+# Determine the bridge-removed graph from the overlap TSV.
+%.bridge-removed.tsv: %.tsv
+	$(time) $(python) $(bin)/physlr bridge-removed --prune-branches=10 --prune-bridges=10 -s0 $< >$@
```

```suggestion
	$(time) $(python) $(bin)/physlr bridge-removal --prune-branches=10 --prune-bridges=10 -s0 $< >$@
```
I know this is a small suggestion -- I just wonder if that is a slightly more clear name

---

yep I agree, especially as the `-` that grammatically (if am right) is needed in between is now hidden behind the common dash we use.

---

am, just a small concern: 
Other functions are named with the type of data they output. For example: `physlr backbone`, `physlr backbone-graph`, `physlr mst`. and in this example we have a bridge-removed graph and that is why I used this naming... while bridge-removal is the action (?). I myself am still feel bad with bridge-removed but do you have alternatives that match name of other functions?
update: we do have functions like `physlr count-molecules`. so `physlr bridge-removal` or `physlr remove-bridges`?

---

Yeah I like `remove-bridges`